pyeudiw/storage/base_storage.py

```diff
@@ -2,7 +2,7 @@ class BaseStorage(object):
     def init_session(self, document_id: str, dpop_proof: dict, attestation: dict):
         NotImplementedError()
 
-    def update_request_object(self, document_id: str, request_object: dict):
+    def update_request_object(self, document_id: str, nonce: str, state: str, request_object: dict):
```

Comment:
state is optional

Reply:
Resolved in commit [9386806](https://github.com/italia/eudi-wallet-it-python/pull/67/commits/9386806d0f517ab0d729b7ac0d79e095f757ff53)